DEFAULT_CONFIG.json5

```diff
@@ -10,37 +10,42 @@
     ////
     ros1: {
       ////
-      //// ROS_MASTER_URI: A URI of the ROS1 Master to connect to, the defailt is http://localhost:11311/
+      //// ros_master_uri: A URI of the ROS1 Master to connect to, the defailt is http://localhost:11311/
       ////
-      // ROS_MASTER_URI: "http://localhost:11311/",
+      // ros_master_uri: "http://localhost:11311/",
 
       ////
-      //// ROS_HOSTNAME: A hostname to send to ROS1 Master, the default is system's hostname
+      //// ros_hostname: A hostname to send to ROS1 Master, the default is system's hostname
       ////
-      // ROS_HOSTNAME: "hostname",
+      // ros_hostname: "hostname",
 
       ////
-      //// ROS_NAME: A bridge node's name for ROS1, the default is "ros1_to_zenoh_bridge" 
+      //// ros_name: A bridge node's name for ROS1, the default is "ros1_to_zenoh_bridge" 
       ////
-      // ROS_NAME: "ros1_to_zenoh_bridge",
+      // ros_name: "ros1_to_zenoh_bridge",
 
       ////
-      //// ROS_BRIDGING_MODE: When to bridge topics:
+      //// ros_bridging_mode: When to bridge topics:
       //// "Automatic"(default) - bridge topics once they are declared locally
       //// "Lazy" - bridge topics once they are declared both locally and required remotely through discovery
       //// Warn: this setting is ignored for local ROS1 clients, as they require a tricky discovery mechanism  
       ////
-      // ROS_BRIDGING_MODE: "Automatic",
+      // ros_bridging_mode: "Automatic",
```

Comment:
In zenoh:
1) config is lowercase
2) *auto* is used instead of *Automatic*

Reply:
fixed